instrumentation/opentelemetry-instrumentation-urllib/src/opentelemetry/instrumentation/urllib/__init__.py

```diff
@@ -171,7 +171,6 @@ def _instrumented_open_call(
 
                 if span.is_recording():
                     span.set_attribute("http.status_code", code_)
-                    span.set_attribute("http.status_text", result.reason)
                     span.set_status(Status(http_status_to_status_code(code_)))
```

Comment:
should the reason be set on the status?

Reply:
Personally I don't think setting a reason phrase adds any value when there is already status code. Copying the same from [RFC](https://tools.ietf.org/html/rfc7230#section-3.1.2). 
> The reason-phrase element exists for the sole purpose of providing a
   textual description associated with the numeric status code, mostly
   out of deference to earlier Internet application protocols that were
   more frequently used with interactive text clients.  A client SHOULD
   ignore the reason-phrase content.

Some places reason is not really http reason phrase it is just formatted exception. I would suggest removing reason phrase on status, WDYT?

Reply:
Ah i see, for some reason I was thinking that `reason` would be more than the status message for the http response